TypeSystem/Serialization/test.cc

```diff
@@ -60,6 +60,14 @@ CURRENT_STRUCT(Serializable) {
   bool operator<(const Serializable& rhs) const { return i < rhs.i; }
 };
 
+CURRENT_STRUCT(Int) {
+  CURRENT_FIELD(x, int32_t, 0);
```

Comment:
@dkorolev,  you seem to repeatedly show me that you don't need my reviews with such "keep Ivan happy" and "whiskey" comments. I'm not a telepath to see you as being drunk so that I don't accidentally interfere with that at the wrong time. Please express yourself clearly and I will unwatch this repo so that we're both happy.